Add Icebreaker post (#299)
DXHeroes/knowledge-base-content: practices/icebreaker.md

---
slug: icebreaker
stages:
  - development

> short_description: Icebreaker is an activity used at the beginning of a meeting to attendees more comfortable with each other, encouraging them to speak up, listen up, join in, and open up to new ideas.

Author: Thanks for the feedback!

  - meetings
  - event
  - rituals
  - agile events
  - scrum
  - retrospective
  - coaching
  - leadership
  - feedback
  - good developer experience
  - demotivated team
keywords:
  - icebreaker
  - activity
  - game
  - break the ice
  - team building
  - communication
  - networking
  - experience exchange
  - remote activity
---

# Icebreaker

**TL;DR**

Icebreaker is an activity used at the beginning of a meeting to attendees more comfortable with each other, encouraging them to speak up, listen up, join in, and open up to new ideas.

## What Is an Icebreaker

An icebreaker is an activity or game designed to "break the ice" between attendees in a meeting. In general, it is used at the beginning of the session to introduce participants to each other, get them relaxed and energized for the upcoming activities. 

Icebreakers are unrestricted to the beginning of a meeting, neither to a random group of attendees. They work very well at the start, middle (after a break), and at the end of a session, and also work very well for attendees who are familiar with each other because they help to set a safe space for attendees to learn and share.

## Why You Might Want Use Icebreakers

> 

Author: Sure.

Icebreakers are fun and remarkable ways to start meetings or training sessions. They are a versatile and adaptable tool used to make the attendees more comfortable with each other, encouraging them to speak up, listen up, join in, and open up to new ideas.

These activities foster networking and a sense of community due to the experience exchange. They can set the tone for the upcoming session and give participants ownership of the learning, with no pressure or high expectations.

## Problems Icebreakers Help to Solve

- Higher anxiety and stress levels pre-meeting
- Low interaction with unknown colleagues
- Low engagement to speak up
- [Demotivated team](/problems/demotivated-team)

> - Decrease in energy levels during a long meeting

Reviewer: ```suggestion
- Decrease in energy levels during a [meaningful meeting](practices/meaningful-meetings)
```

Author: I added 
```suggestion
- Decrease in energy levels during non [meaningful meetings](practices/meaningful-meetings)
```
because we are talking about the **problems** icebreakers help to solve. What do you think about this approach?

- Unpleasant meetings
- [Toxic Team Culture](/problems/toxic-team-culture)

## How to Implement Icebreakers?

> One of the best places to find an icebreaker is online, but we must be careful because they usually are associated with adjectives that may not describe its essence.  When looking for an icebreaker, we must have in mind the goal we want to achieve, the time we can spend with it, and if it will help or not to achieve the goal.

Reviewer: Please, consider changing subject we into you. 
Something alongside this change:
You should be careful since they usually have misleading descriptions. (but we must be careful because they usually are associated with adjectives that may not describe its essence)

Author: Sure. I also improved this paragraph to make it clearer.

> Above, we present some examples of how to choose icebreakers aiming for a specific goal.

Author: Sorry, sometimes I switch above and bellow :grimacing:. Will fix it.

> - If the goal is to introduce participants to each other so they can interact several times during the meeting, the chosen activity must focus on the individual presentation plus establish a bonding way between attendees. For example, by making them recognize and appreciate differences and similarities in each other.

Author: I agree with you.

tags:
  - meaningful meetings
  - meetings
  - event
  - rituals
  - agile events
  - scrum
  - retrospective
  - coaching
  - leadership
  - feedback
  - good developer experience
  - demotivated team
keywords:
  - icebreaker
  - activity
  - game
  - break the ice
  - team building
  - communication
  - networking
  - experience exchange
  - remote activity
---

# Icebreaker

**TL;DR**

An icebreaker is an activity you can use at the beginning of a meeting to make attendees more comfortable with each other, encouraging them to speak up, listen up, join in, and open up to new ideas.

## What Is an Icebreaker?

An icebreaker is an activity or game designed to "break the ice" between attendees in a meeting. In general, it is used at the beginning of the session to introduce participants to each other, get them relaxed and energized for the upcoming activities. 

Icebreakers are unrestricted to the beginning of a meeting, neither to a random group of attendees. You can use one at the start, during, and at the end of the session. Icebreakers also help with setting a safe space for attendees (even for familiar ones) to learn and share. 

## Why You Might Want Use Icebreakers?

> ![Icebreakers example](../files/icebreakers.jpg)  

Reviewer 2: ```suggestion
![Icebreakers example](/files/icebreakers.jpg)  
```

[Icebreakers example](https://business.linkedin.com/talent-solutions/blog/recruiting-tips/2016/9-icebreakers-that-every-recruiter-can-use)

Icebreakers are fun and remarkable ways to start meetings or training sessions. They are a versatile and adaptable tool used to make the attendees more comfortable with each other, encouraging them to speak up, listen up, join in, and open up to new ideas.

These activities foster networking and a sense of community due to the experience exchange. They can set the tone for the upcoming session and give participants ownership of the learning, with no pressure or high expectations.

## Problems Icebreakers Help to Solve

- Higher anxiety and stress levels pre-meeting
- Low interaction with unknown colleagues
- Low engagement to speak up
- [Demotivated team](/problems/demotivated-team)

> - Decrease in energy levels during non [meaningful meetings](practices/meaningful-meetings)

Reviewer 2: ```suggestion
- Decrease in energy levels during non [meaningful meetings](/practices/meaningful-meetings)
```
